### PR TITLE
Add compilation help

### PR DESCRIPTION
**Pull request by tamplan**
Use of submodule implies some commands before a compilation attempt.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -18,6 +18,12 @@ See https://wiki.gnome.org/Projects/SessionManagement/
 Installation
 ============
 
+In order to build this program from the source code you need git installed and
+ a working MATE environment version 1.2, with the development tools installed
+properly.
```
Comment (raveit65) [inline README:23]: 1.2 ???
Where is this from? 
Damned, i over read that in mate-terminal PR.....

Comment (tamplan) [reply README:23]: Damned too, copy/paste error... I need to modify.

Comment (tamplan) [reply README]: Updated

Comment (raveit65) [reply README]: A tip, `git log -p` will show you the commit in terminal after commiting localy.

Comment (tamplan) [reply README]: Thanks, this could help.

**Pull request merged**